ska_conda/pkg_defs/ska3-flight/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - ska.matplotlib ==3.11.2
     - ska.parsecm ==3.3.1
     - ska_path ==3.1
+    - ska_sync ==0.1.1
```

Comment:
This has git tag 0.1.1 and version 0.1 in the code (as far as I can tell) which should probably get cleaned up at some point.